kubernetes/README.md

```diff
@@ -0,0 +1,31 @@
+# Creating a Kubernetes Deployment
+
+## Summary
+
+The application configuration is in the `config.yaml` Kubernetes secret,
+which stores it in the secret `issecret`.
```

Comment:
Which what stores? I think this clause is unnecessary. People interested in kubernetes will understand this all from the first half of the sentence.